.gitignore

```diff
@@ -13,3 +13,4 @@ examples/services/metalv1/create_device_in_metro/create_device_in_metro
 examples/services/metalv1/plans/plans
 examples/services/ordersv2/get_order_details/get_order_details
 examples/services/smarthandsv1/get_smarthands_types/get_smarthands_types
+examples/services/lookupv2/get_patch_panels/get_patch_panels
```

Comment:
Non-blocking comment:  I hadn't noticed these new `.gitignore` lines before.  Based on the example added in this PR, it doesn't look like this line is needed (I don't see the `examples/services/lookupv2/get_patch_panels` example writing any files that would match the `examples/services/lookupv2/get_patch_panels/get_patch_panels` path).  Are these lines something we can safely clean up in a later PR?